packages/discord.js/typings/index.d.ts

```diff
@@ -4112,6 +4113,7 @@ export interface ChatInputApplicationCommandData extends BaseApplicationCommandD
   descriptionLocalizations?: LocalizationMap;
   type?: ApplicationCommandType.ChatInput;
   options?: ApplicationCommandOptionData[];
+  nsfw?: boolean;
```

Comment:
Upstream has the valid types for this as "all". I think this should belong in the base, then?